src/ankit/Scanner/EachCharacterFrequency.java

```diff
@@ -0,0 +1,31 @@
+package ankit;
+//Assignment16: To find the frequency of each character from user given word.
+import java.util.Scanner;
+
+public class EachCharacterFrequency {
+	
+	void scannerOperation(String word, int length) {
+		
+		for(int index =0; index<length; index++) {
+			int counter=0;
+			char character =word.charAt(index);
+			for(int charIndex=0; charIndex<length; charIndex++) {
+				if(character == word.charAt(charIndex))
+					counter++;
+			}
+			System.out.println(character+ " is present " +counter+ " times in String");
+		}
```

Comment:
static void scannerOperation(String word) {

		for (int index = 0; index < word.length(); index++) {
			int counter = 0;
			char character = word.charAt(index);
			for (int charIndex = 0; charIndex < word.length(); charIndex++) {
				if (character == word.charAt(charIndex))
					counter++;
			}
			System.out.println(character + " is present " + counter + " times in String");
		}
	}